posthog/api/person.py

```diff
@@ -357,6 +360,8 @@ def destroy(self, request: request.Request, pk=None, **kwargs):
             # Once the person is deleted, queue deletion of associated data, if that was requested
             if "delete_events" in request.GET:
                 self._queue_event_deletion(person)
+            if "delete_recordings" in request.GET:
+                self._queue_delete_recordings(person)
```

Comment:
i am very boring when it comes to backends

we should have a test on this :)

we call

`http://localhost:8010/api/person/4165233d-0a3c-51a5-b400-a385c3b7843c?delete_events=false&delete_recordings=true`

and assert  `temporal.start_workflow` is called appropriately is probably enough